The default WINAPI_FAMILY is "desktop app", triggering a false positive
here, which causes Win8 APIs (e.g. CreateFile2) to be used.

--- origsrc/zlib-1.2.8/contrib/minizip/iowin32.c	2013-04-28 17:57:11.000000000 -0500
+++ src/zlib-1.2.8/contrib/minizip/iowin32.c	2013-07-26 00:49:38.042212400 -0500
@@ -26,7 +26,7 @@
 #endif
 
 
-#if defined(WINAPI_FAMILY_PARTITION) && (!(defined(IOWIN32_USING_WINRT_API)))
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8) && (!(defined(IOWIN32_USING_WINRT_API)))
 #if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
 #define IOWIN32_USING_WINRT_API 1
 #endif
